pandas/tests/extension/base/accumulate.py

```diff
@@ -0,0 +1,43 @@
+import warnings
+
+import pytest
+
+import pandas as pd
+import pandas.util.testing as tm
+
+from .base import BaseExtensionTests
+
+
+class BaseAccumulateTests(BaseExtensionTests):
+    """
+    Accumulation specific tests. Generally these only
+    make sense for numeric/boolean operations.
+    """
+
+    def check_accumulate(self, s, op_name, skipna):
+        result = getattr(s, op_name)(skipna=skipna)
+        expected = getattr(s.astype("float64"), op_name)(skipna=skipna)
+        tm.assert_almost_equal(result, expected, check_dtype=False)
```

Comment:
I believe this is copied from the other tests, but what do people think about structuring things a bit differently? This is doing a lot of stuff, making it hard to customize for a specific type.

So I'd recommend something like

```python
def get_expected(self, s, op_name, skipna):
    return getattr(s.astype("float64"), op_name)(skipna=skipna)
```

and then the test can do the `self.assert_series_equal(result, expected)`